README.md

```diff
@@ -53,42 +80,65 @@ When you ask your assistant to “Turn on the light”, it will use the auth bea
     "automation" :
     {
      "name": "automation",
-     "url": "https://YOUR-URL/openhabGoogleAssistant"
+     "url": "https://YOUR-OPENHAB-CLOUD-URL/openhabGoogleAssistant"
     }
   }
 }
 ```
 
-* Afterwards upload this action file using the following command:
+* Afterwards deploy this action file using the following command:
 ```
-gactions update --action_package action.json --project <YOUR-GOOGLE-CLOUD-PROJECT-ID>
+gactions update --action_package action.json --project <YOUR-APPLICATION>
 ```
 
-Google Assistant will call the service endpoint: `https://YOUR-URL/openhabGoogleAssistant`.
+Google Assistant will call the service endpoint: `https://YOUR-OPENHAB-CLOUD-URL/openhabGoogleAssistant`.
 This web service will receive parameters (intents) from Google and will query/modify openHAB items through openHAB-cloud depending on those parameters.
 
 * You need to Add "App information”, including name and account linking details to the Actions Console
 * Afterwards please run the following command in the gaction CLI:
 ```
-gactions test --action_package PACKAGE_NAME --project AGENT_ID
+gactions test --action_package action.json --project <YOUR-APPLICATION>
 ```
 
-### Account Linkage & OAuth2:
+## Setup your Database
+* SSH into to your openhab cloud instance
+* Open the mongodb client `mongo` and enter these commands
+```
+use openhab
+db.oauth2clients.insert({ clientId: "<CLIENT-ID>", clientSecret: "<CLIENT SECRET>"})
```

Comment:
@holmes  I found out that if you add the keys for:
icon: xxxx.png (must be in the public/img/oauth2/ folder on OH-Cloud instance), I found an image online
name: "Google Assistant"
description: "Access to openHAB Cloud specific for Actions on Google Assistant."

Full Example:
db.oauth2clients.insert({ clientId: "<CLIENT-ID>", clientSecret: "<CLIENT SECRET>", "icon" : "<USER-IMAGE.EXT>", "homeUrl" : "<LINK TO API HOME>", "name" : "<LINK API NAME>", "description" : "<DESCRIPTION/REASON FOR OAUTH>" }

It will then populate correctly within OH Cloud -> Applications tab (see screenshot).  It's more cosmetic, but maybe good to have documented somewhere??

![image](https://user-images.githubusercontent.com/32886013/35926147-7f1094e6-0bf5-11e8-8290-c1501b96cf7d.png)

Reply:
@skimdankish thanks for the tip, i'll udpate it.